packages/extension/src/components/DaSettings.vue

```diff
@@ -22,6 +22,11 @@
       <da-switch label="Card animations" class="small settings__animations"
                  :checked="enableCardAnimations" @toggle="toggleCardAnimations"/>
     </div>
+    <div class="settings__column">
```

Comment:
This is actually a pretty nice hack to implement the HTML part, but I think we should go with a "flex" based solution here.
I had in mind `flex-wrap` although it will be much harder to implement it with the same look that you achieved here (which is the right one, obviously). What do you think?

Reply:
Okay, I'll give flex-wrap a try. Also, I didn't change the variable name because of consistency purposes, since we have named the variable a `showOnlyNotReadPosts` in the backend so I wanted to keep it the same in frontend as well because variable names of other settings are also same in frontend and backend.